src/jdk.compiler/share/classes/com/sun/tools/javac/code/Type.java

```diff
@@ -2021,6 +2049,26 @@ public boolean isNullOrReference() {
         public <R, P> R accept(TypeVisitor<R, P> v, P p) {
             return v.visitTypeVariable(this, p);
         }
+
+        public boolean isUniversal() {
+            return universal;
+        }
+
+        @Override
+        public Type withTypeVar(Type t) {
```

Comment:
Uhm - isn't this method used on wildcard types, and it all other types is supposed to just return `this` ?